components/ui/AppFiltersResultsLayout.vue

```diff
@@ -44,10 +44,8 @@ export default class AppFiltersResultsLayouts extends Vue {
       margin-bottom: $layout-01;
     }
 
-    &_2-cols-xlg {
-      @include mq($from: x-large) {
-        height: calc(100% - #{$layout-02});
-      }
```

Comment:
As pointed out in https://github.com/Qiskit/qiskit.org/pull/1797#discussion_r638681699